.github/workflows/testspace.yml

```diff
@@ -0,0 +1,25 @@
+name: Testspace
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Function name'
+        required: true
+      payload:
+        description: 'Function input-payload'
+        required: true
+      context:
+        description: 'Function execution-context'
+        required: true
+env:
+  IN_NAME:    ${{ github.event.inputs.name }}
+  IN_PAYLOAD: ${{ github.event.inputs.payload }}
+  IN_CONTEXT: ${{ github.event.inputs.context }}
+jobs:
+  fixture:
```

Comment:
As it is expected to have multiple fixtures, you should better name this job as `setup`